From 9931b6e4c6b8cfa8f72a83386ee4f9acf4a7bd45 Mon Sep 17 00:00:00 2001
From: hari gowtham <hgowtham@redhat.com>
Date: Tue, 27 Oct 2015 11:51:33 +0530
Subject: [PATCH 60/67] tier: Typo while setting the wrong value of low/hi watermark

While setting the wrong value of watermark-hi/low the output
shows "compatiblevalue" whereas it should be "compatible value"

Change-Id: I29c8f9a954928d22e436465f4ebc30bd08640138
BUG: 1275155
Signed-off-by: hari gowtham <hgowtham@redhat.com>
Reviewed-on: http://review.gluster.org/12432
Reviewed-by: Humble Devassy Chirammal <humble.devassy@gmail.com>
Tested-by: Gluster Build System <jenkins@build.gluster.com>
Reviewed-by: Dan Lambright <dlambrig@redhat.com>
Signed-off-by: Hari Gowtham <hgowtham@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/60329
Tested-by: Dan Lambright <dlambrig@redhat.com>
---
 xlators/mgmt/glusterd/src/glusterd-volume-set.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/xlators/mgmt/glusterd/src/glusterd-volume-set.c b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
index 7ab8734..4099f5c 100644
--- a/xlators/mgmt/glusterd/src/glusterd-volume-set.c
+++ b/xlators/mgmt/glusterd/src/glusterd-volume-set.c
@@ -80,8 +80,9 @@ validate_tier (glusterd_volinfo_t *volinfo, dict_t *dict, char *key,
         if (strstr (key, "watermark-hi") ||
             strstr (key, "watermark-low")) {
                 if ((origin_val < 1) || (origin_val > 99)) {
-                        snprintf (errstr, sizeof (errstr), "%s is not a compatible"
-                                  "value. %s expects a percentage from 1-99.",
+                        snprintf (errstr, sizeof (errstr), "%s is not a "
+                                  "compatible value. %s expects a "
+                                  "percentage from 1-99.",
                                   value, key);
                         gf_msg (this->name, GF_LOG_ERROR, EINVAL,
                                 GD_MSG_INCOMPATIBLE_VALUE, "%s", errstr);
-- 
1.7.1

